### PR TITLE
Fix job crashing for 9 time-bin FST data

### PR DESCRIPTION
**Pull request by techuan-huang**
Issue #631 was due to an inconsistent number of time bins used in the data and the codes.
In most FST data and in the codes, we used 3 time bins, while some data used 9 time bins.
Only changing a constant in `StRoot/StEvent/StFstConsts.h` is needed to fix this issue.

### CODE REVIEW

**Comment by genevb** [comment]
Thanks, @techuan-huang . Two notes from me:
- I tried my test job with this patch and it ran to completion for me without crashing. I was careful to rebuild all the libraries that depend on this one file.
- If this passes review, we should tag `SL23f_1` and I will rebuild the library. `SL23f` has not been used for any other productions besides the preview production yet, so we should be fine with this patch.

-Gene

**Review by plexoos — APPROVED** [review]
If the tests confirm that the issues is fixed then it is fixed

**Review by jdbrice — APPROVED** [review]

**Comment by genevb** [comment]
Are we waiting for @akioogawa 's approval, or can this be merged now?

**Pull request merged**
